packages/engine/src/game/functions/functionsActions.ts

```diff
@@ -17,7 +17,7 @@ import { getEntityFromRoleUuid, getGame, getGameEntityFromName, getRole, getUuid
 // TODO: create enum actions
 const gameActionComponents = {
   'HaveBeenInteracted': HaveBeenInteracted,
-  'HasHadCollision': HasHadCollision
+  'HasHadCollision': GameObjectCollisionTag
```

Comment:
whoops, i must have missed this

Reply:
i re-work this stuff, you see it in new PR